SendEmail/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Communication.Email;
+using Azure.Communication.Email.Models;
+
+namespace SendEmail
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            // This code demonstrates how to send email using Azure Communication Services.
+            var connectionString = "<ACS_CONNECTION_STRING>";
+            var emailClient = new EmailClient(connectionString);
+            var subject = "Send email quick start - dotnet";
+            var emailContent = new EmailContent(subject)
+            {
+                PlainText = "This is plain mail send test body \n Best Wishes!!",
+                Html = "<html><body><h1>Quick send email test</h1><br/><h4>Communication email as a service mail send app working properly</h4><p>Happy Learning!!</p></body></html>"
+            };
+            var sender = "<SENDER_EMAIL>";
+
+            EmailRecipients emailRecipients = new EmailRecipients(new List<EmailAddress> {
```

Comment:
Sorry, I missed these. Anything that is = new XXXX should be var.

line 29 also